dogpile/cache/backends/memcached.py

```diff
@@ -17,6 +18,8 @@
 from ..api import NO_VALUE
 from ... import util
 
+log = logging.getLogger(__name__)
```

Comment:
**mike bayer** (zzzeek) wrote:

this should be removed

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/dogpile.cache/+/2965